packages/ocsfml/ocsfml.2.0/opam

```diff
@@ -15,7 +15,8 @@ synopsis: "Binding to the C++ SFML gaming library."
 description: """
 The C++ library SFML (2.0) must be installed before any installation of this binding.
 See http://www.sfml-dev.org/index.php for more information."""
+dev-repo: "git+https://github.com/JoeDralliam/Ocsfml.git"
 url {
-  src: "https://github.com/KyoDralliam/Ocsfml/tarball/master"
+  src: "https://github.com/KyoDralliam/Ocsfml/archive/5a90ae9596861166bf69a36423318924595cd172.tar.gz"
```

Comment:
2021-09-14 11:36.52: Error in ocsfml.2.0: Failed to download the archive. Details: https://github.com/KyoDralliam/Ocsfml/archive/5a90ae9596861166bf69a36423318924595cd172.tar.gz (Bad checksum, expected md5=bf823023c9dcc9de9b41ec8eb77861cb)

Reply:
I checked with pkgdiff and the content is unchanged, I'll update the checksum.

Reply:
Thanks!